flink-runtime/src/main/java/org/apache/flink/runtime/operators/sort/NormalizedKeySorter.java

```diff
@@ -47,61 +47,61 @@
 	
 	public static final int MAX_NORMALIZED_KEY_LEN_PER_ELEMENT = 8;
 	
-	private static final int MIN_REQUIRED_BUFFERS = 3;
+	public static final int MIN_REQUIRED_BUFFERS = 3;
 	
-	private static final int LARGE_RECORD_THRESHOLD = 10 * 1024 * 1024;
+	public static final int LARGE_RECORD_THRESHOLD = 10 * 1024 * 1024;
 	
-	private static final long LARGE_RECORD_TAG = 1L << 63;
+	public static final long LARGE_RECORD_TAG = 1L << 63;
 	
-	private static final long POINTER_MASK = LARGE_RECORD_TAG - 1;
+	public static final long POINTER_MASK = LARGE_RECORD_TAG - 1;
 
```

Comment:
The reason `public` is used here because `Janino` first check accessibility of these variables and it seems not able to access them when `protected` is used and it throws the error below.
```
org.codehaus.commons.compiler.CompileException: Field "LARGE_RECORD_THRESHOLD" is not accessible

	at org.codehaus.janino.ReflectionIClass$ReflectionIField.getConstantValue(ReflectionIClass.java:340)
	at org.codehaus.janino.UnitCompiler.getConstantValue2(UnitCompiler.java:4433)
	at org.codehaus.janino.UnitCompiler.access$10000(UnitCompiler.java:182)
	at org.codehaus.janino.UnitCompiler$11.visitFieldAccess(UnitCompiler.java:4407)
	at org.codehaus.janino.Java$FieldAccess.accept(Java.java:3229)
```

Reply:
Maybe put that into the comment inside the code?